BioinformaticTools-inator/BioinformaticToolsInator.py

```diff
@@ -0,0 +1,90 @@
+from typing import Dict, List, Tuple, Union
+from modules import dna_rna_funcs, fastq_funcs
+
+
+def run_dna_rna_tools(
+        *args: str
+        ) -> Union[List[str], Dict[str, Dict[str, int]], str]:
+    """
+    Performs a specified action on all sequences transmitted from input
+
+    Args:
+    - DNA or RNA sequences as str and a procedure to perform
+      as the last argument
+
+    Possible procedures:
+    - Main procedures: reverse, transcribe, complement, reverse_complement
+    - Additional procedures:
+      nucleotide_frequency, gc_content, find_start_codons
+
+    Returns:
+    - Union[List[str], Dict[str, Dict[str, int]], str]:
+      sequence or a collection of sequences with paramters
+    """
+    *seqs, operation = args
+    for seq in seqs:
+        dna_rna_funcs.is_valid_na(seq)
+    if operation == 'reverse':
+        return dna_rna_funcs.reverse(*seqs)
+    elif operation == 'transcribe':
+        return dna_rna_funcs.transcribe(*seqs)
+    elif operation == 'complement':
+        return dna_rna_funcs.complement(*seqs)
+    elif operation == 'reverse_complement':
+        return dna_rna_funcs.reverse_complement(*seqs)
+    elif operation == 'nucleotide_frequency':
+        return dna_rna_funcs.nucleotide_frequency(*seqs)
+    elif operation == 'gc_content':
+        return dna_rna_funcs.gc_content(*seqs)
+    elif operation == 'find_start_codons':
+        return dna_rna_funcs.find_start_codons(*seqs)
+    else:
+        print('Unidentified instruction')
```

Comment:
Мне кажется, что вместо принта, лучше поставить еще один return. В него можно прописать как строку, говорящую об ошибке, так в будущем самому прописать полноценную ошибку. Но тогда получившую конструкцию можно будет еще упростить. От этого: 

if условие:
    return a
else:
    return b 

мы перейдем к этому: 

if условие:
    return a
return b 

Потому что функция доходит до первого return и дальше не идет. Так что долой else! Сделаем код более читаемым ;)
```suggestion
    return "Unidentified instruction"
```

Reply:
Лучше только такое штуки еще бы оформить как код